distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -33,5 +32,6 @@ To import previously exported configuration settings:
 ** importing from a system installed on a different OS
 ** importing from a system installed in a different directory location
 * To keep the export/import process simple and consistent, all system configuration files are required to be under the `${home_directory}` directory and not be softlinks. Presence of external files or symbolic links during export will not fail the export; they will yield warnings. It will be up to the adminsitrator to manually copy these files over to the new system before proceeding with the import. The import process will verify their presence and consistency and yield warnings if they don't match the original files.
-* The import process will re-import all configurations done on the original system as part of the <<_hardening_checklist,hardening process>> including changes to starting scripts and certificates.
+* The import process will restore all applications, features, and/or bundles to the same state (i.e. installed, uninstalled, started, stopped, ...) they were in originally. It is therefore not require to go through the standard installation process after unzipping the ${branding} distribution.
```

Comment:
I did that in the upcoming PR where we are introducing this as an option on the command instead of making it mandatory. I guess I can add a quick note for now here.